### DISCUSSION TITLE
Hoarder limits and syncing flow

### DISCUSSION THREAD

**Discussion by DaanMich** [sunflower-land/sunflower-land#2421] [Ideas]
Hey everyone

I think recently some people have been having not such a great user experience with the hoarder limits. 

**A recent example:** have 3 block bucks left but hit the carrot hoarder limit. This personally discouraged me from going for the giant carrot 🥕 

**Possible solutions:**

- make sure hoarder limits support the full 5 block bucks
-  Reduce the nr of block bucks that can be bought as to support the idea of more frequent syncing
- Make the syncing progress a fixed x days 

**Bumpkin statistics issue**
To add to the previous problem, storing progress on chain costs a lot due to bumpkin statistics. 

We need a separate button to store bumpkin statistics on chain. 

Since bumpkin data on chain isn’t usable anymore for dune (storing progress on chain happens less often) I do not see the point of forcing people to sync these statistics on chain. 

Let me know your thoughts down below 👇

**Comment by future2back** [comment]
Do you have the new seed stock, my friend? I have never reached planting limit during the 5 syncs.

**Reply by DaanMich** [reply]
I do not but to my knowledge this would only up the seed hoarder limits and not the actual crops harvested hoarder limits which is what I’m struggling with. 

Imo the issue lies with the crop hoarder limits not being supported for 5 block bucks. 

If the team wants people to sync after x days then imo they should just make it a fixed nr of days until you are forced to sync. Way better user experience imo. 

That’s just my honest opinion tho. Could be missing some elements as to why the 5 block bucks are not supporting hoarder limits

**Comment by vitt0** [comment]
Adding a option to buy 3 Bucks would be a good middle ground for the middle to higher end farms that can hit some of the hoarder limits pretty easilty if they use the 5 bucks option, even more since they added the Toolshed and the warehouse.

I also agree about the store on chain separation from the bumpkin statistics, at the moment they are only making this problem with the hoarder limits be more tedious 😞

**Reply by DaanMich** [reply]
Yup, that would definitely also work! 

I think 2 block bucks would be ideal. 

Prime example would be the screenshot showing the hoarder limits added below. This is with 2/5 BB’s used (huge thanks for the amazing tool!!). 

I’ve seen a lot of people who just went back to using 1 BB at a time which just defeats the ease of use imo (and also cuts into the devs pockets since 5 BB’s get them more fees). 

For anyone reading this and unaware: https://sflplus.info/

![1792458E-F17C-4B53-B42F-A3504F61E6C1](https://user-images.githubusercontent.com/111318826/232250567-7bed5c08-9f82-4082-8052-6cf04d5cddce.jpeg)

**Reply by vitt0** [reply]
Yep at the moment I decided to go back to use the 1 buck option since I build the toolshed, I mostly focus in chopping wood so for me is either use the 5 bucks options and have to be dealing with the hoarder limits or use the 1 buck option and waste Matic in gas.

I would rather be paying 0.40$ to buy 3 Bucks and give more money to the project that wasting extra Matic in gas.

**Reply by vitt0** [reply]
Gwei around 120-150

0.238~ MATIC ---- $0.28 USD -- 1st Sync
0.137~ MATIC ---- $0.16 USD -- 2nd Sync
0.120~ MATIC ----$0.14 USD -- 3th Sync

I just try this, first sync with all the seeds that I normally use at 0~, after syncing I used the buck and bought all the seeds that I use + axes and sand shovels and sync again for $0.16, then I used this buck and sync again for only $0.14 

So now I have full seeds + full stock of seeds to buy + 1 buck without use

Looking only at the gas that I paid in the first sync was $0.17, so buying buying 5 bucks would be $0.92~ giving a total of $0.184 per buck

With my 3 txn I finish paying $0.58 so $0.193~ per Buck, but like this I have to deal doing a pretty annoying hassle and in the end the project only got $0.30 when they could have got maybe $0.40

Now that I think about it, maybe a 2 bucks option like you mention would be also good for $0.25
